src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DTF = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateStart = LocalDate.parse(dateFrom, DTF);
+        LocalDate dateFinish = LocalDate.parse(dateTo, DTF);
+        int salary = 0;
+        StringBuilder builder = new StringBuilder("Report for period ");
+        builder.append(dateFrom).append(" - ").append(dateTo);
```

Comment:
If you have a chain of method invocation, it's a good practice to call each method on a new line.

This formatting style makes it clear that each method call is separate and helps visually distinguish them. It can also make it easier to add or remove method calls if needed, as each call is on its own line.